report_pre_thesis/src/sections/experiments.tex

```diff
@@ -1,26 +1,115 @@
+\centering
+\begin{tabular}{llll}
+\hline
+p-value    & Student's & Welch's & Wilcoxon \\
+\hline
+PLS1-SM    & 67.71\% & 84.04\% & 73.53\% \\
+ICA        & 32.38\% & 71.38\% & 64.06\% \\
+MOC        & 54.69\% & 75.45\% & 100.00\% \\
+\hline
+\end{tabular}
+\caption{p-values of the Student's t-test, Welch's t-test, and Wilcoxon signed-rank test for the RMSEs between the original and our replicas of the PLS1-SM, ICA, and MOC models, respectively.}
+\label{tab:results_ttests}
+\end{table}
+
+\subsection{Chosen Experiments}\label{sec:chosen_experiments}
+To evaluate the performance of each of the components in the pipeline, we have conducted the following series of experiments.
+
+\begin{enumerate}
+	\item\label{enum:experiment1} No outlier removal for PLS1-SM to evaluate the significance of our automated outlier removal process and whether outlier removal is necessary.
+	\item\label{enum:experiment2} Maintain the threshold values from the second iteration of the outlier removal in the PLS1-SM training process to see if a more or less conservative approach is advantageous.
+	\item\label{enum:experiment3} MAD for ICA to evaluate the significance of the outlier removal step in the ICA phase.
+	\item\label{enum:experiment4} Evaluating the impact on ICA model performance when using all five location datasets versus just one.
+	\item\label{enum:experiment5} Other models to compare the performance of the PLS1-SM and ICA models to other models.
+	% \item PLS outlier removal on ICA (?) instead of MAD.
+\end{enumerate}
 
 \noindent
-Given that the original authors did not perform experiments using alternative methods to demonstrate the efficacy of their chosen approach, this omission results in a lack of comprehensive understanding regarding the full potential of the pipeline's performance.
-While they did perform hyperparameter tuning, they did not conduct experiments using different outlier removal methods or alternative models.
-This raises questions about the optimality of the chosen methodology, as a comparative analysis with different methodologies could reveal superior approaches.
-Experimenting with alternative methods means that we can uncover which components contribute the most to the overall error and therefore would benefit the most from further research and development.
-Should a substitution of a component within the pipeline with an alternative method yield improved outcomes, it would indicate that the currently employed method represents a limitation in the overall pipeline, thus highlighting an area that necessitates enhancement.
+We have chosen these experiments based on the following reasons.
+For Experiment~\ref{enum:experiment1}, we are interested in evaluating the significance of our automated outlier removal process due to its differences from the original manual process.
+Additionally, we are interested in examining whether outlier removal is necessary for the PLS1-SM phase by comparing the results with and without outlier removal.
```

Comment:
I think perhaps this is a less interesting reason. One could argue that the original authors did outlier removal because they found it necessary.

I think we should instead focus on trying to figure out what impact our outlier detection has on the results and hence why it makes sense to do a test without outlier removal. Essentially, I would like to just see more of the topic from the first paragraph.

```suggestion
By conducting an experiment without outlier detection we will be able to discern the benefit of our outlier removal approach.
Additionally, this experiment could also yield insights into the benefit of the original, manual outlier detection approach and help determine whether an automated approach can lead to better predictions.
```

Something like that.

Reply:
We do both